xmtp_mls/src/groups/mod.rs

```diff
@@ -1,30 +1,37 @@
 mod intents;
 
+#[cfg(test)]
+use std::println as debug;
```

Comment:
There's also this crate, which allows for log statements to get printed in tests https://crates.io/crates/test-env-log

Reply:
So adding this (or more specifically the updated `test-log` crate) seems to cause half of the `encrypted_store` tests to fail - it looks like there's a conflict in how the logger is set up with one of our other dependencies. Might just skip the hassle for now

Reply:
Fine by me